stepup/.env

```diff
@@ -0,0 +1,8 @@
+#AZUREMFA_PHP_IMAGE=php82-apache2-node20-composer2:latest
+#DEMOGSSP_PHP_IMAGE=php82-apache2-node20-composer2:latest
```

Comment:
What is the difference between these images? But if it is meant that we CAN use different images for different projects, I understand. For now they seem the same

Reply:
That's correct, you can override it on a per project basis.